### ISSUE TITLE
Need Extrinsic calibration?

### ISSUE THREAD

**Issue by d2h7** [dvorak0/VI-MEAN#8]
Hi! 
i am wondering does it necessary to have extrinsic calibration matrix, i don't find any file for extrinsic calibration, but when i replace the topic with my own sensors , it seems that the imu doesn't align with vision,  the path is wrongly calculated.
i am waiting for your reply, thank you!

**Comment by dvorak0** [comment]
Yes. For details, see https://github.com/dvorak0/VI-MEAN/blob/release/self_calibration_estimator/src/parameters.cpp and https://github.com/dvorak0/VI-MEAN/blob/release/self_calibration_estimator/launch/mapping/sample.launch.

**Comment by d2h7** [comment]
Thank you for your reply!
i notice the ric_ is related to the extrinsic rotation matrix, but i don't quite understand what does y0,p0,r0 means, does it means the rotation angle of x, y, z?

**Comment by dvorak0** [comment]
Euler angle. here is the convert function. https://github.com/dvorak0/VI-MEAN/blob/release/self_calibration_estimator/src/utility.h

**Comment by d2h7** [comment]
i see…thank you very much!
